extensions/amp-story-panning-media/validator-amp-story-panning-media.protoascii

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-story-panning-media
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-story-panning-media"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-story-panning-media>
+  html_format: AMP
+  tag_name: "AMP-STORY-PANNING-MEDIA"
+  requires_extension: "amp-story-panning-media"
+  mandatory_parent: "AMP-STORY-GRID-LAYER"
```

Comment:
Does this mean your new component has to be a direct child of grid-layer? Can you remind us why we need to enforce this, and why it couldn't be any descendant of grid-layer?

Reply:
Let's let it be any descendant for now. I need to test layouts to see if this would actually enforce any best practices.